src/main/java/org/broadinstitute/hellbender/tools/copynumber/utils/TagGermlineEvents.java

```diff
@@ -0,0 +1,81 @@
+package org.broadinstitute.hellbender.tools.copynumber.utils;
+
+import org.broadinstitute.barclay.argparser.Argument;
+import org.broadinstitute.barclay.argparser.BetaFeature;
+import org.broadinstitute.barclay.argparser.CommandLineProgramProperties;
+import org.broadinstitute.hellbender.cmdline.ExomeStandardArgumentDefinitions;
+import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
+import org.broadinstitute.hellbender.cmdline.programgroups.CopyNumberProgramGroup;
+import org.broadinstitute.hellbender.engine.GATKTool;
+import org.broadinstitute.hellbender.tools.copynumber.formats.collections.CalledCopyRatioSegmentCollection;
+import org.broadinstitute.hellbender.tools.copynumber.utils.annotatedregion.SimpleAnnotatedGenomicRegion;
+import org.broadinstitute.hellbender.tools.copynumber.utils.annotatedregion.SimpleAnnotatedGenomicRegionCollection;
+import org.broadinstitute.hellbender.tools.copynumber.utils.germlinetagging.SimpleGermlineTagger;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.List;
+
+@CommandLineProgramProperties(
+        oneLineSummary = "(EXPERIMENTAL) Do a simplistic tagging of germline events in a tumor segment file.",
```

Comment:
`summary` should precede `oneLineSummary`, and neither should end in a period.  I think it's better to limit them both to one line if possible.  Also, statements like "The algorithm used is very simplistic" are probably not necessary in the summaries.  Further description can go in the Javadoc for the tools.

Reply:
By the way, none of your tools have Javadoc or command-line examples.  @sooheelee Are we requiring this for experimental tools?

Reply:
@sooheelee I'm fine with writing docs and CLI examples.  These are only two tools.  I just started these so long ago that I forgot to add the docs and CLI examples later.

Reply:
Also, these are not listed as `@DocumentedFeature`